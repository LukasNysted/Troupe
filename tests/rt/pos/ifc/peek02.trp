let fun f () = 
      let val x = peek (0, `{}`, `{}`) 
          val y = peek (1, `{}`, `{}`)
      in print (x,y) 
      end 
    val p = spawn f 
in send (p, 1) ;
   (if true raisedTo `{secret}` then send (p,2) else ()); 
   blockdecl (authority);
   send (p, 3)
end 
