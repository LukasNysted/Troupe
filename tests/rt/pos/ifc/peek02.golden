2021-08-09T20:19:04.968Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: ()@{}%{}
((1, "<local>"), (3, "<local>"))
